src/careers/index.html

```diff
@@ -79,46 +79,48 @@ <h1 class="h3">Newest Openings</h1>
     <div class="block
                 block__flush-top
                 block__flush-bottom">
-        <div class="media
-                    block
-                    block__flush-top
-                    block__flush-bottom
-                    block__padded-bottom">
+        <section class="media
+                        block
+                        block__flush-top
+                        block__flush-bottom
+                        block__padded-bottom">
             <div class="media_image-container
                         media_image-container__wide-margin">
                 <img class="media_image media_image__150 u-centered-on-mobile"
-                     src="/static/img/clipboard-round-300x300.png">
+                     src="/static/img/clipboard-round-300x300.png"
+                     alt="">
```

Comment:
In both cases, the image is decorative. A description of the image isn't helpful to someone using a screenreader.